functions-and-scope.js

```diff
@@ -68,8 +117,15 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe kan ik iedere waarde van de array langsgaan?
 // * Op welke conditie moet ik checken?
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan opslaan?
+// variabele hoogsteCijfer aanmaken en die steeds aanpassen ALS je binnen de array een hoger cijfer tegenkomt
 // Log het antwoord in de terminal.
-
+let hoogsteCijfer = 0;
```

Comment:
In deze opdracht is het prima om als default waarde 0 te pakken, maar stel je eens voor dat hier een array binnenkomt als [-1, -4, -2, -3]. In dit voorbeeld is de hoogste waarde -1, maar met jouw functie zou er 0 uit komen terwijl die niet eens in de array voorkomt.

Nogmaals, het maakt voor deze situatie niet uit, maar het is goed om je te realiseren dat wanneer operaties als deze gaat doen op een array, dat je niet een default waarde pakt, maar altijd de eerste waarde van de array, dan weet je zeker dat je goed zit namelijk:
```suggestion
let hoogsteCijfer = grades[0];
```